internal/services/resource/resource_arm_template_deployment_test.go

```diff
@@ -45,7 +45,7 @@ func TestAccTemplateDeployment_requiresImport(t *testing.T) {
 				check.That(data.ResourceName).ExistsInAzure(r),
 			),
 		},
-		data.RequiresImportErrorStep(r.requiresImport),
+		// data.RequiresImportErrorStep(r.requiresImport), // A bug? It expects an error but there's not an error
```

Comment:
Do you know why there's supposed to be an error here?

Reply:
It is supposed to appear an error that it needs to be imported first so it can be managed but this is no the case, probably is the same case of this [test](https://github.com/abner-dou/terraform-provider-azurestack/blob/d5fd5281d79c616a81446e25d5e131eb0de73570/internal/services/storage/storage_account_resource_test.go#L50)